android_beta/8.0beta.yml

```diff
@@ -287,8 +292,100 @@ notes:
 
   - note: >
       Cross-device and cross-app import is now limited to Thunderbird, as it was intended for a
-      way to migrate K-9 and Thunderbir Desktop users to Thunderbird for Android.
+      way to migrate K-9 and Thunderbird Desktop users to Thunderbird for Android.
     short_note: Cross-device import limited to Thunderbird
     k9mail_only: true
     tag: fixed
     group: 4
+
+  ## Beta 5
+  # New
+  - note: >
+      The unread message colors have been updated to make it more apparent when a message is
+      unread.
+    short_note: Unread message colors are updated for better visibility.
```

Comment:
Just describing the changes. Not suggesting wording for the changelog:
- The text color of message previews in the message list has been changed to be lighter (for both read and unread messages)
- The text color of read(!) messages in the message list has been changed to be lighter.

(That's from the perspective of the light theme. When using the dark theme: s/lighter/darker/).

Reply:
It seems to me what you describe can be summarized as it is now, "Unread message colors are updated for better visibility". Is anything missing?

Reply:
The text color used for unread messages hasn't changed. The one for read messages has.